### ISSUE TITLE
[ISSUE] 关于api服务，在利用chatTTS合成音频指令，会在固定时间点卡住

### ISSUE THREAD

**Issue by YanyanPop** [lenML/Speech-AI-Forge#211]
### 确认清单

- [x] 我已经阅读过 README.md 和 dependencies.md 文件
- [x] 我已经确认之前没有 issue 或 discussion 涉及此 BUG
- [x] 我已经确认问题发生在最新代码或稳定版本中
- [x] 我已经确认问题与 API 无关
- [x] 我已经确认问题与 WebUI 无关
- [x] 我已经确认问题与 Finetune 无关

### 你的issues

环境：Linux ubuntu11.4
python版本：conda环境 python3.10.16
运行过程：
**默认采用wav格式**
curl http://localhost:7870/v1/audio/speech   -H "Authorization: Bearer anything_your_wanna"   -H "Con
    "model": "chattts",
    "input": "Today is a wonderful day to build something people love! [lbreak]",
    "voice": "female2",
    "style": "chat"
  }'   --output speech.wav
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  222k    0  222k  100   155  17386     11  0:00:14  0:00:13  0:00:01 46535
——————————————————————————————————————————————————
python launch.py
fatal: No names found, cannot describe anything.
2025-03-12 01:37:40,147 - datasets - INFO - PyTorch version 2.5.1 available.
2025-03-12 01:37:40,936 - modules.core.models.tts.CosyVoiceModel - INFO - No CosyVoice model found
2025-03-12 01:37:41,218 - modules.devices.devices - INFO - Using half precision: torch.float16
2025-03-12 01:37:41,219 - modules.devices.devices - INFO - Using device: cuda
2025-03-12 01:37:41,596 - modules.api.Api - INFO - Registered API: GET /v1/ping
2025-03-12 01:37:41,597 - modules.api.Api - INFO - Registered API: GET /v1/versions
2025-03-12 01:37:41,597 - modules.api.Api - INFO - Registered API: GET /v1/audio_formats
2025-03-12 01:37:41,598 - modules.api.Api - INFO - Registered API: GET /v1/models/reload
2025-03-12 01:37:41,599 - modules.api.Api - INFO - Registered API: GET /v1/models/unload
2025-03-12 01:37:41,599 - modules.api.Api - INFO - Registered API: GET /v1/models/list
2025-03-12 01:37:41,600 - modules.api.Api - INFO - Registered API: GET /v1/styles/list
2025-03-12 01:37:41,602 - modules.api.Api - INFO - Registered API: GET /v1/speakers/list
2025-03-12 01:37:41,603 - modules.api.Api - INFO - Registered API: POST /v1/speakers/refresh
2025-03-12 01:37:41,604 - modules.api.Api - INFO - Registered API: POST /v1/speakers/update
2025-03-12 01:37:41,605 - modules.api.Api - INFO - Registered API: POST /v1/speaker/create
2025-03-12 01:37:41,606 - modules.api.Api - INFO - Registered API: POST /v1/speaker/update
2025-03-12 01:37:41,608 - modules.api.Api - INFO - Registered API: POST /v1/speaker/detail
2025-03-12 01:37:41,616 - modules.api.Api - INFO - Registered API: GET /v1/tts
2025-03-12 01:37:41,617 - modules.api.Api - INFO - Registered API: POST /v1/ssml
2025-03-12 01:37:41,619 - modules.api.Api - INFO - Registered API: POST /v1/text:synthesize
2025-03-12 01:37:41,620 - modules.api.Api - INFO - Registered API: POST /v1/speech:recognize
2025-03-12 01:37:41,620 - modules.api.Api - INFO - Registered API: POST /v1/speech:longrunningrecognize
2025-03-12 01:37:41,622 - modules.api.Api - INFO - Registered API: POST /v1/audio/speech
2025-03-12 01:37:41,627 - modules.api.Api - INFO - Registered API: POST /v1/audio/transcriptions
2025-03-12 01:37:41,628 - modules.api.Api - INFO - Registered API: POST /v1/prompt/refine
2025-03-12 01:37:41,630 - modules.api.Api - INFO - Registered API: POST /v1/text/normalize
2025-03-12 01:37:41,630 - modules.api.Api - INFO - Registered API: GET /v1/xtts_v2/speakers
2025-03-12 01:37:41,631 - modules.api.Api - INFO - Registered API: POST /v1/xtts_v2/tts_to_audio
2025-03-12 01:37:41,632 - modules.api.Api - INFO - Registered API: GET /v1/xtts_v2/tts_stream
2025-03-12 01:37:41,633 - modules.api.Api - INFO - Registered API: POST /v1/xtts_v2/set_tts_settings
2025-03-12 01:37:41,644 - modules.api.Api - INFO - Registered API: POST /v1/stt/transcribe
2025-03-12 01:37:41,653 - modules.api.Api - INFO - Registered API: POST /v1/stt/stream
2025-03-12 01:37:41,658 - modules.api.Api - INFO - Registered API: POST /v1/vc
2025-03-12 01:37:41,660 - modules.api.Api - INFO - Registered API: POST /v2/tts
2025-03-12 01:37:41,661 - modules.api.api_setup - INFO - allow CORS origin: *
INFO:     Started server process [880936]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:7870 (Press CTRL+C to quit)
2025-03-12 01:38:57,429 WETEXT INFO found existing fst: /home/gxy/anaconda3/envs/speech_ai_forge/lib/python3.10/site-packages/tn/en_tn_tagger.fst
2025-03-12 01:38:57,429 - wetext-en_normalizer - INFO - found existing fst: /home/gxy/anaconda3/envs/speech_ai_forge/lib/python3.10/site-packages/tn/en_tn_tagg
2025-03-12 01:38:57,429 WETEXT INFO                     /home/gxy/anaconda3/envs/speech_ai_forge/lib/python3.10/site-packages/tn/en_tn_verbalizer.fst
2025-03-12 01:38:57,429 - wetext-en_normalizer - INFO -                     /home/gxy/anaconda3/envs/speech_ai_forge/lib/python3.10/site-packages/tn/en_tn_verb
2025-03-12 01:38:57,430 WETEXT INFO skip building fst for en_normalizer ...
2025-03-12 01:38:57,430 - wetext-en_normalizer - INFO - skip building fst for en_normalizer ...
2025-03-12 01:38:58,234 - modules.core.models.tts.ChatTTS.ChatTTS - INFO - Loading ChatTTS models
2025-03-12 01:38:58,255 - modules.repos_static.ChatTTS.ChatTTS.core - INFO - try to load from local: ./models/ChatTTS
2025-03-12 01:38:58,255 - modules.repos_static.ChatTTS.ChatTTS.core - INFO - checking assets...
2025-03-12 01:38:59,180 - modules.repos_static.ChatTTS.ChatTTS.core - INFO - all assets are already latest.
2025-03-12 01:38:59,550 - modules.repos_static.ChatTTS.ChatTTS.core - INFO - vocos loaded.
2025-03-12 01:38:59,746 - modules.repos_static.ChatTTS.ChatTTS.core - INFO - dvae loaded.
2025-03-12 01:39:03,596 - modules.repos_static.ChatTTS.ChatTTS.core - INFO - gpt loaded.
2025-03-12 01:39:03,851 - modules.repos_static.ChatTTS.ChatTTS.core - INFO - decoder loaded.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
2025-03-12 01:39:03,865 - modules.repos_static.ChatTTS.ChatTTS.core - INFO - tokenizer loaded.
2025-03-12 01:39:03,887 - modules.core.models.tts.ChatTTS.ChatTTS - INFO - ChatTTS models loaded
code:  11%|█████████████████▌
2025-03-12 01:39:09,256 - modules.core.handler.encoder.StreamEncoder - INFO - StreamEncoder header set, channels: 1, sample_width: 2, sample_rate: 24000

会在这里卡住，没有反应，返回wav格式的音频文件，但没有任何内容

**转化为mp3格式**
 curl http://localhost:7870/v1/audio/speech   -H "Authorization: Bearer anything_your_wanna"   -H "Con
    "model": "chattts",
    "input": "Today is a wonderful day to build something people love! [lbreak]",
    "voice": "female2",
    "style": "chat", "response_format":"mp3"
  }'   --output speech.mp3
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 38625    0 38445  100   180   2286     10  0:00:18  0:00:16  0:00:02 10688
——————————————————————————————————————————————————
2025-03-12 01:42:14,766 - modules.repos_static.ChatTTS.ChatTTS.core - INFO - tokenizer loaded.
2025-03-12 01:42:14,788 - modules.core.models.tts.ChatTTS.ChatTTS - INFO - ChatTTS models loaded
code:  11%|█████████████████
2025-03-12 01:42:20,111 - modules.core.handler.encoder.StreamEncoder - INFO - StreamEncoder header set, channels: 1, sample_width: 2, sample_rate: 24000

会在同样的地方卡住，每次都卡在这个地方，麻烦大佬帮忙看看可能是哪里存在问题

**Comment by zhzLuke96** [comment]
无法复现，请提供更详细的内容，应该包括 是否使用最新代码、webui是否可用、是否有其他报错、并提供输出的文件

这里不一定是卡住，这个进度条显示的是最大 token 数，每次生成不一定会用完最大token

**Comment by YanyanPop** [comment]
2025-03-12 08:40:34,102 - modules.core.models.tts.ChatTTS.ChatTTS - INFO - ChatTTS models loaded
code:  11%|████████████▍                                                                                                     | 223/2048(max) [00:04, 45.32it/s]
2025-03-12 08:40:39,988 - modules.core.handler.encoder.StreamEncoder - INFO - StreamEncoder header set, channels: 1, sample_width: 2, sample_rate: 24000
在此之后无输出了，没有看到其他的报错信息。
webui可用，网页版可以正常生成音频；没有其他任何报错了。目前修改了服务端口号并增加了人声增强的模型，甚至不出现token的进度条直接生成.wav/.mp3的文件了；文件生成比网页版快很多，wav文件大小222KB（针对上述文本输出大小），播放点开无进度条。

**Comment by zhzLuke96** [comment]
1. 你测试的适合没有进度条可能是缓存了，相同的请求不会重复生成
2. 默认格式输出的wav文件文件头是空标记，这样是为了提供流式的效果，依赖播放器能力，比如 potplayer 无法支持，但是 chrome 浏览器可以支持
3. 如果需要保存文件，请设置 format

如果还是有问题，方便的话请把你说的有问题的文件上传上来

**Comment by YanyanPop** [comment]
感谢，已解决，chrome浏览器打开是ok的
